Description: add build flags from CPPFLAGS
Author: Wookey <wookey@debian.org>

--- a/Makefile
+++ b/Makefile
@@ -748,6 +748,7 @@
 OPTIMIZE = -fdata-sections -ffunction-sections -O3
 DEBUGFLAGS = -g3
 PKG_CONFIG_LIBS := $(shell $(PKG_CONFIG) --cflags $(LIBS_ALL))
+CXXFLAGS += $(CPPFLAGS)
 CXXFLAGS += $(DEBUGFLAGS) $(DEFINES) $(OPTIMIZE) $(PKG_CONFIG_LIBS) -MP -MMD -pthread -Wall -Wshadow -Wsuggest-override -std=c++17
 CFLAGS = $(filter-out -Wsuggest-override, $(filter-out -std=%,$(CXXFLAGS))) -std=c99
 LDFLAGS += -lm -lpthread
