Description: add build flags from CPPFLAGS
Author: Wookey <wookey@debian.org>

--- a/Makefile
+++ b/Makefile
@@ -590,7 +590,8 @@
 
 OPTIMIZE = -fdata-sections -ffunction-sections
 DEBUGFLAGS = -g3
-CXXFLAGS += $(DEBUGFLAGS) $(DEFINES) $(OPTIMIZE) $(shell $(PKG_CONFIG) --cflags $(LIBS_ALL)) -MP -MMD -pthread -Wall -Wshadow -std=c++17 -O3
+CXXFLAGS +=$(CPPFLAGS)
+CXXFLAGS +=$(DEBUGFLAGS) $(DEFINES) $(OPTIMIZE) $(shell $(PKG_CONFIG) --cflags $(LIBS_ALL)) -MP -MMD -pthread -Wall -Wshadow -std=c++17 -O3
 CFLAGS = $(filter-out -std=%,$(CXXFLAGS)) -std=c99
 LDFLAGS += -lm -lpthread
 GLIB_COMPILE_RESOURCES = $(shell $(PKG_CONFIG) --variable=glib_compile_resources gio-2.0)
