Description: add build flags from CPPFLAGS
Author: Wookey <wookey@debian.org>

--- a/Makefile
+++ b/Makefile
@@ -797,6 +797,7 @@ OPTIMIZE = -fdata-sections -ffunction-sections -O3
 DEBUGFLAGS = -g3
 WARNFLAGS = -Wall -Wshadow
 PKG_CONFIG_LIBS := $(shell $(PKG_CONFIG) --cflags $(LIBS_ALL))
+CXXFLAGS += $(CPPFLAGS)
 CXXFLAGS += $(DEBUGFLAGS) $(DEFINES) $(OPTIMIZE) $(PKG_CONFIG_LIBS) -MP -MD -pthread $(WARNFLAGS) -std=c++17
 CFLAGS = $(filter-out -Wsuggest-override, $(filter-out -std=%,$(CXXFLAGS))) -std=c99
 LDFLAGS += -lm -lpthread -lstdc++
