Description: add build flags from CPPFLAGS
Author: Wookey <wookey@debian.org>

--- a/Makefile
+++ b/Makefile
@@ -719,6 +719,7 @@ LIBS_ALL = $(LIBS_COMMON) gtkmm-3.0 epoxy cairomm-pdf-1.0 librsvg-2.0 libzmq lib
 
 OPTIMIZE = -fdata-sections -ffunction-sections
 DEBUGFLAGS = -g3
+CXXFLAGS += $(CPPFLAGS)
 CXXFLAGS += $(DEBUGFLAGS) $(DEFINES) $(OPTIMIZE) $(shell $(PKG_CONFIG) --cflags $(LIBS_ALL)) -MP -MMD -pthread -Wall -Wshadow -Wsuggest-override -std=c++17 -O3
 CFLAGS = $(filter-out -Wsuggest-override, $(filter-out -std=%,$(CXXFLAGS))) -std=c99
 LDFLAGS += -lm -lpthread
